src/lib.rs

```diff
@@ -1260,6 +1288,13 @@ impl Connection {
                 ack_elicited = true;
             }
 
+            // If the packet this frame belongs to is an early Application one,
+            // buffer the frame for later processing.
+            if !self.is_established() && hdr.ty == packet::Type::Application {
```

Comment:
nit: you have a reverse order of logic on https://github.com/cloudflare/quiche/pull/68/files#diff-b4aea3e418ccdb71239b96952d9cddb6R1256, is there some reason for that?

Reply:
No reason. Changed this to match the other.